PageViewController.swift

```diff
@@ -0,0 +1,211 @@
+        return pageControl
+    }()
+    var currentIndex: Int {
+        guard let vc = viewControllers?.first else { return 0 }
+        return pages.firstIndex(of: vc) ?? 0
+    }
+    var getStarted = UIButton()
+    let skipButton = UIButton()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.view.backgroundColor = .white
+        
+        self.dataSource = self
+        self.delegate = self
+
+        configurePageControl()
+
+        configureButtons()
+    }
+    
+    override func viewDidLayoutSubviews() {
+        pageControl.subviews.forEach {
+            $0.transform = CGAffineTransform(scaleX: 1.5, y: 1.5)
+        }
+    }
+    
+    func configureButtons() {
+        print("Button should show")
+        let nextButton = UIButton()
+        nextButton.setTitle("Next", for: .normal)
+        nextButton.setTitleColor(.gray, for: .normal)
+        nextButton.frame = CGRect(x: 30, y: 100, width: 50, height: 30)
+        view.addSubview(nextButton)
+        nextButton.addTarget(self, action: #selector(nextClicked(_:)), for: .touchUpInside)
+        nextButton.translatesAutoresizingMaskIntoConstraints = false
+        nextButton.centerYAnchor.constraint(equalTo: pageControl.centerYAnchor).isActive = true
+        nextButton.rightAnchor.constraint(equalTo: view.safeAreaLayoutGuide.rightAnchor, constant: -24).isActive = true
+        nextButton.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor).isActive = true
+        
+        
+        skipButton.setTitle("Skip", for: .normal)
+        skipButton.setTitleColor(.gray, for: .normal)
+        skipButton.frame = CGRect(x: 30, y: 100, width: 50, height: 30)
+        view.addSubview(skipButton)
+        skipButton.addTarget(self, action: #selector(self.skipClicked(_:)), for: .touchUpInside)
+        skipButton.translatesAutoresizingMaskIntoConstraints = false
+        skipButton.centerYAnchor.constraint(equalTo: pageControl.centerYAnchor).isActive = true
+        skipButton.leftAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leftAnchor, constant: 24).isActive = true
+        skipButton.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor).isActive = true
+        
+        let closeButton = UIButton()
+        closeButton.setTitle("X", for: .normal)
+        closeButton.setTitleColor(.gray, for: .normal)
+        closeButton.frame = CGRect(x: 30, y: 100, width: 50, height: 30)
+        view.addSubview(closeButton)
+        closeButton.addTarget(self, action: #selector(self.close(_:)), for: .touchUpInside)
+        closeButton.translatesAutoresizingMaskIntoConstraints = false
+        closeButton.leftAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leftAnchor, constant: 24).isActive = true
+        closeButton.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor).isActive = true
+        
+        
+        getStarted.setImage(UIImage(named: "getStarted"), for: .normal)
```

Comment:
I think we should just make this button in code instead of using the image, since it just has text and rounded corners